### ISSUE TITLE
Chrome uses system title bar and borders when i opened it from Cairo and it looks weird

### ISSUE THREAD

**Issue by denizjcan** [cairoshell/cairoshell#541]
When I open Google Chrome (or chrome related shortcuts) from Quick launch, Programs menu or dynamic desktop, it uses the system title and look weird (black frame/blue tabs etc).

I reinstalled Chrome and Cairo (even deleted files in appdata). I checked Chrome in regedit, it looks normal. 
(Computer\HKEY_CLASSES_ROOT\ChromeHTML\shell\open\command => "C:\Program Files\Google\Chrome\Application\chrome.exe" --single-argument %1)
I also checked the Chrome shortcuts in Cairo and didn't see a problem. Even when I started Chrome with the bat file in cairo, the same problem occurred. I have no problem opening the same shortcuts from the windows desktop, taskbar or start menu. If I open Chrome from the desktop, I have no problem when I open a new window from Cairo.

I added a screen recording to explain it more clearly.
https://user-images.githubusercontent.com/22716602/110348966-42fd6900-8043-11eb-8dce-9509b7acac27.mp4

Steps to reproduce
1. Open Google chrome from Quick launch/programs menu/dynamic desktop

Expected behavior
Starting Google Chrome with normal window borders.

System information:
 - Windows version: Windows 10 20H2
 - Cairo version: 0.4.76.45508 - Pre-release
 - 32-bit or 64-bit: 64 bit
 - In Cairo Settings > Advanced, is Cairo set as shell?: No, it is not set as shell.

**Comment by dremin** [comment]
Oh, that is strange. Thanks for the recording, it helps! I'll look into this.

**Comment by denizjcan** [comment]
> Oh, that is strange. Thanks for the recording, it helps! I'll look into this.

Thank you :).

**Comment by dremin** [comment]
If you go to `%localappdata%\Google\Chrome` and rename the User Data folder (to something like "User Data1", so you can revert the change afterwards) then open Chrome, does it use the correct window appearance? It should appear like a fresh installation.

**Comment by denizjcan** [comment]
I tried but unfortunately the same problem still persists. 
Such a problem surprised me because I don't think there will be any connection between the problem and Cairo. Maybe a system file in Windows got corrupted but I don't have this problem anywhere else except Cairo.

**Comment by dremin** [comment]
When you followed those steps was the result a fresh Chrome profile that still had the same issue?

**Comment by denizjcan** [comment]
Yes, when I renamed the folder and reopened chrome it opened a chrome that looks like newly installed without my account, addons, bookmarks etc. However, the problem remained.

**Comment by dremin** [comment]
Which version of Chrome are you currently running?

**Comment by denizjcan** [comment]
89.0.4389.82 (64 bit).

**Comment by dremin** [comment]
I have not been able to reproduce this on that version. One thought, does the issue persist if you open Windows Settings > Personalization > Colors, then toggle the "title bars and window borders" setting?

**Comment by denizjcan** [comment]
Yes, the issue also persist with this method. I think I'm hopeless at this point 😃 .
I have wrote this post by thinking that the problem is relevant to Cairo, but I see that the solutions you offer is outside Cairo. For this reason I think it would be to steal your time to want more solutions from you. Thank you very much for all your help. I wish you success in Cairo and your other works 😄.

**Comment by dremin** [comment]
A change went in recently that I think may have improved this situation. Are you still seeing this behavior with the latest version of Cairo?
